pkg/venafi/cloud/connector.go

```diff
@@ -537,16 +544,16 @@ func (c *Connector) searchCertificatesByFingerprint(fp string) (*CertificateSear
 
 */
 type managedCertificate struct {
-	Id                         string `json:"id"`
-	CompanyId                  string `json:"companyId"`
-	LatestCertificateRequestId string `json:"latestCertificateRequestId"`
-	CertificateName            string `json:"certificateName"`
+	Id                   string `json:"id"`
+	CompanyId            string `json:"companyId"`
+	CertificateRequestId string `json:"certificateRequestId"`
+	//CertificateName            string `json:"certificateName"`
```

Comment:
can we safely remove this commented code?

Reply:
done